### PR TITLE
Add API function and outsourced api key with .env

### LINKED ISSUE #5: Build "API".ts
#### Outsource necessary functions for the API in a .ts file 

Create all necessary functions in a *API*.ts to outsource them. 

#### Sample
[Sample](https://github.com/lmachens/photoplay/pull/49/commits/330d451ffb3053d77c89431820fc60d3af3ca7cc)

#### Tasks

- [x] Create BASE_URL
- [x] Create fetch function
- [ ] Create specific functions using the fetch function

### PR DESCRIPTION
**Pull request by roabram**
fixes #5

### CODE REVIEW

**Review by Dev-dfm — APPROVED** [review]
lgtm 👍

**Review by Niklas-Schaefers — APPROVED** [review]
lgtm 🚀

**Review by J-Bannach — APPROVED** [review]
lgtm 🍰

File: .env [hunk 1]
```diff
@@ -1 +1,2 @@
-PORT=3333
\ No newline at end of file
+PORT=3333
+API_KEY=08f8ca3c51313dd913f7f765a26ebc29
```
Comment (lmachens) [inline .env]: Don't forget to ignore this file 🙈

File: src/server/routes.ts [hunk 2]
```diff
@@ -1,9 +1,20 @@
...
 
 const router = express.Router();
 
-router.get('/crypto', (_req, res) => {
-  res.status(404).send('Hello World');
+router.get('/singleStock', async (_req, res, next) => {
+  try {
+    const { query } = _req;
+    if (typeof query.stock !== 'string' || typeof query.time !== 'string') {
+      res.status(404).send('Hello World');
```
Comment (lmachens) [inline src/server/routes.ts:10]: Hello World?

File: samples.http [hunk 3]
```diff
@@ -1,11 +1,11 @@
 @port = {{$dotenv PORT}}
 @baseUrl = http://localhost:{{port}}
 
+
 ### Single Crypto 
-GET {{baseUrl}}/api/crypto
+GET /api/singleStock(AAPL)
```
Comment (lmachens) [inline samples.http]: I think this route is incorrect. `(AAPL)` looks weird

**Review by lmachens — COMMENTED** [review]
Good job. A deployment would be nice to test the API

File: src/server/routes.ts (hunk 2 above)
Comment (roabram) [reply src/server/routes.ts:10]: Was still there from testing. Changed it now.

**Pull request merged**